bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionParameter.java

```diff
@@ -81,6 +83,8 @@
     private String unitLabel;
 
     private String context;
+
+    @SerializedName("default")
```

Comment:
Interesting. I am actually not sure whether this should rather be considered a bug. The deserialization should imho produce DTOs, which we should then map the the internal classes.
@danchom, @maggu2810, what do you think?